contracts/test/foundry/FleekERC721/AccessControl.t.sol

```diff
@@ -214,10 +215,12 @@ contract Test_FleekERC721_AccessControl is Test_FleekERC721_Base, Test_FleekERC7
         CuT.setTokenENS(tokenId, ens);
 
         // TokenOwner
+        transferENS(ens, tokenOwner);
         vm.prank(tokenOwner);
         CuT.setTokenENS(tokenId, ens);
 
         // TokenController
+        transferENS(ens, tokenController);
         vm.prank(tokenController);
         CuT.setTokenENS(tokenId, ens);
```

Comment:
Should we not allow changing by controller? This can just be set by owner right?

Reply:
The way it is right now it is equals to the other properties that controllers may change. In my opinion is not bad that way because if someone else than the owner of the NFA is owner of the ENS, the owner of the NFA can add the owner of the ENS as a controller and he can call `setTokenENS` with his account. The other situation would be the owner of the ENS create a subdomain for the owner of the NFA, both ways works.

Reply:
Alright that's fair. I like the idea of the owner being able to add the owner of the ENS as a controller :+1: